composer.json

```diff
@@ -11,7 +11,8 @@
     "swaggest/php-code-builder": "^0.2.41"
   },
   "require-dev": {
-    "pestphp/pest": "^2.33"
+    "pestphp/pest": "^2.33",
+    "mockery/mockery": "^1.6.7"
```

Comment:
this can be removed

Reply:
Thanks! You'll also need to run `composer remove mockery/mockery` to also remove them from the `composer.lock` file

Reply:
Ah. I was unaware this won't happen on it's own. Thanks for the remark. Done now.